governance/polls/Add GUNIV3DAIUSDCFT with 0.01% Fee Tier as a new Vault Type - November 29, 2021.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Add GUNIV3DAIUSDCFT with 0.01% Fee Tier as a new Vault Type - November 29, 2021
+summary: This poll proposes system parameters which could be used to initialize GUNI-DAI-USDC-FT as a new vault type.
+discussion_link: https://forum.makerdao.com/t/g-uni-dai-usdc-0-01-tier-onboard-g-uni-for-univ3s-new-0-01-fee-tier/11738
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Risk Variable
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-11-29T16:00:00
+end_date: 2021-12-02T16:00:00
+---
+# Poll: Add GUNIV3DAIUSDCFT with 0.01% Fee Tier as a new Vault Type - November 29, 2021
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the MakerDAO mandated actors. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, November 29 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding GUNIV3DAIUSDCFT (Gelato DAI-USDC) with 0.01% Fee Tier as a new vault type in the Maker Protocol with the parameters below:
+
+### Initial Collateral Parameters
+
+* Underlying Collateral: GUNI-DAI/USDC
```

Comment:
Maybe clarify this is the 0.01% ilk here

"Underlying Collateral: GUNI-DAI/USDC (0.01% fee)" - not 100% sure though @LongForWisdom any thoughts?

Reply:
Yes "GUNI-DAI/USDC (0.01% fee)" is good.

Reply:
Good :) Thanks! Fixed it!